tests/cases/fourslash/fourslash.ts

```diff
@@ -133,6 +134,7 @@ declare namespace FourSlashInterface {
         quickInfoIs(expectedText?: string, expectedDocumentation?: string): void;
         quickInfoExists(): void;
         typeDefinitionCountIs(expectedCount: number): void;
+        implementationCountIs(expectedCount: number): void;

```

Comment:
This is only used once to assert that there are 0 implementations. There's not much value to being able to write something like `verify.not.implementationCountIs(3)`; we want to test exactly what those implementations are.